ddtrace/tracer.py

```diff
@@ -1013,6 +1016,8 @@ def shutdown(self, timeout=None):
             for processor in chain(span_processors, SpanProcessor.__processors__):
                 if hasattr(processor, "shutdown"):
                     processor.shutdown(timeout)
+            if config._data_streams_enabled:
```

Comment:
Is it ever possible that `config._data_streams_enabled` could be set false during runtime, causing us to incorrectly skip this shutdown? If possible, should we instead just `try except` around the shutdown?